engine/resources/cron.go

```diff
@@ -145,11 +146,18 @@ func (obj *CronRes) makeComposite() (*FileRes, error) {
 	path := fmt.Sprintf("/etc/systemd/system/%s.timer", obj.Name())
 	if obj.Session {
 		// user timer
-		path = fmt.Sprintf("%s/.config/systemd/user/%s.timer", os.Getenv("HOME"), obj.Name())
+		u, err := user.Current()
+		if err != nil {
+			return nil, errwrap.Wrapf(err, "error getting current user")
+		}
+		if u.HomeDir == "" {
+			return nil, fmt.Errorf("user has no home directory")
+		}
+		path = fmt.Sprintf("%s/.config/systemd/user/%s.timer", u.HomeDir, obj.Name())
```

Comment:
In this case, keep the sprintf for the suffix, but use path.Join for the homedir+prefix to avoid double slashes and stuff.